contracts/contracts/vault/VaultAdmin.sol

```diff
@@ -185,18 +193,86 @@ contract VaultAdmin is VaultStorage {
             ];
             allStrategies.pop();
 
+            // Mark the strategy as not supported
+            strategies[_addr].isSupported = false;
+
             // Withdraw all assets
             IStrategy strategy = IStrategy(_addr);
             strategy.withdrawAll();
+
             // Call harvest after withdraw in case withdraw triggers
             // distribution of additional reward tokens (true for Compound)
             _harvest(_addr);
             emit StrategyRemoved(_addr);
-            // Mark the strategy as not supported
-            strategies[_addr].isSupported = false;
         }
     }
 
+    /**
+     * @dev Add a swap token to the tokens that get liquidated for stablecoins
+     *      whenever swap is called. The token must have a valid feed registered
+     *      with the price provider.
+     * @param _addr Address of the token
+     */
+    function addSwapToken(address _addr) external onlyGovernor {
+        for (uint256 i = 0; i < swapTokens.length; i++) {
+            if (swapTokens[i] == _addr) {
+                revert("Swap token already added");
+            }
+        }
+
+        // Revert if feed does not exist
+        IOracle(priceProvider).price(_addr);
+
+        swapTokens.push(_addr);
+
+        // Give Uniswap infinte approval
+        if (uniswapAddr != address(0)) {
+            IERC20 token = IERC20(_addr);
+            token.safeApprove(uniswapAddr, 0);
+            token.safeApprove(uniswapAddr, uint256(-1));
+        }
+
+        emit SwapTokenAdded(_addr);
+    }
+
+    /**
+     * @dev Remove a swap token from the tokens that get liquidated for stablecoins.
+     * @param _addr Address of the token
+     */
+    function removeSwapToken(address _addr) external onlyGovernor {
+        bool added = false;
+        for (uint256 i = 0; i < swapTokens.length; i++) {
+            if (swapTokens[i] == _addr) {
+                added = true;
```

Comment:
supernit: can add a break here to exit the loop and maybe save a tiny bit of gas.